### PR TITLE
Replaces "yiff"

### PR DESCRIPTION
**Pull request by Tsar-Salat**
## About The Pull Request

Replaces usages of "yiff".

Owospeak is a funny thing for adminbus and Ive given it in the past, but it shouldnt force players to yell about gratuitous sexual congress by accident. That would probably be a headache of an admin report, and be kinda gross for the players in question.

Why did I even have to make this pr, this is a stain on my profile

## Why It's Good For The Game

image

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Do I need to

</details>

## Changelog
:cl:
code: adjusts a diseases speech replacement
/:cl:

### CODE REVIEW

**Comment by PatienceStPim** [comment]
might I suggest replacing it with "nuzzle" instead so its still funny when people beg to be killed

File: strings/owo_talk.json [hunk 1]
```diff
@@ -23,10 +23,10 @@
...
         "lizard": "scalie",
         "damn": "mew",
-        "kill": "yiff",
-        "killed": "yiffed",
-        "murder": "yiff",
-        "murdered": "yiffed",
+        "kill": "keel",
+        "killed": "keeled",
+        "murder": "mworder",
+        "murdered": "mworder",
```
Comment (Rukofamicom) [inline strings/owo_talk.json]:
```suggestion
        "kill": "nuzzle",
        "killed": "nuzzled",
        "murder": "nuzzle",
        "murdered": "nuzzled",
```

Comment (Tsar-Salat) [reply strings/owo_talk.json]: Fine. If it leads to my admintools getting fixed quicker.

**Review by Rukofamicom — APPROVED** [review]
This is now suitably cringey

**Pull request merged**